scripts/gen-notice.sh

```diff
@@ -134,4 +133,4 @@ npm ls --omit=dev --all --parseable \
                 }
             })
         })
-    ' >$OUTFILE
+    ' >>$OUTFILE
```

Comment:
Reviewer note: This fixes a small bug in this script. This PR is the first time `gen-notice.sh` is being used without the `--lint` argument, so this overwriting of earlier content written to the "OUTFILE" wasn't noticed.